frontend/src/components/BtoDisplayItemsOverview.tsx

```diff
@@ -0,0 +1,32 @@
+import {BtoDisplayItem} from "../model/BtoDisplayItem";
+import BtoDisplayItemCard from "./BtoDisplayItemCard";
+import BtoUserItemCard from "./BtoUserItemCard";
+
+type BtoDisplayItemsOverviewProps = {
+    btoDisplayItems: BtoDisplayItem []
+}
+
+export default function BtoDisplayItemsOverview({btoDisplayItems} : BtoDisplayItemsOverviewProps){
+    return(
+        <div>
+            <h1><u>Übersichtsseite</u></h1>
+            {btoDisplayItems.map(item => <BtoDisplayItemCard btoDisplayItem={item} />)}
+            <hr/><hr/>
+            <h1><u>Beispiel User Seite (hier z.B. User "U11")</u></h1>
+            <h1>Darstellung eigener Aktionen:</h1>
+            {btoDisplayItems.filter((item) => (item.category === "action"  &&  item.actionOwner === "U11"))
```

Comment:
Ich finde diesen `filter` mit `map` ganz raffiniert.

Manche meiner Kollegen würden hier wahrscheinlich empfehlen den Code aus dem JSX-Teil heraus, nach oben direct in die Function zu ziehen - der Übersichtlichkeit wegen. Ist aber Geschmackssache: ich persönlich finde es so noch gut lesbar.

Reply:
Danke für das Lob, ich werde es für den Prototyp auch so lassen